0004-post-tags.md

```diff
@@ -0,0 +1,299 @@
+- Feature Name: Post Tags
+- Start Date: 2023-11-04
+- RFC PR: [LemmyNet/rfcs#0004](https://github.com/LemmyNet/rfcs/pull/4)
+- Lemmy Issue: [LemmyNet/lemmy#317](https://github.com/LemmyNet/lemmy/issues/317)
+
+# Summary
+
+Ability to add descriptive tags to Posts similar to Reddit's Flair system primarily on a per-community basis.
+
+# Motivation
+
+Post Tags would allow for vastly improved content filtering (and to a lesser extent searching) both for Users and Admins/Moderators. 
+
+# Guide-level explanation
+
+Tags grant you more fine grained control over the content you see on Lemmy. Certain users do not want to see certain content and tags allow you to do exactly that.
+
+For example a lot of communities offer mixed content such as Memes, News and Discussions. Some people would like to only see News or Discussions. In such a case they could blacklist (filter out) the "Meme" tag, going forward they would not be presented with any Memes in that community.
+
+Another use for tags is better handling of how posts are displayed. For example some people would like the content of a post to stay hidden for all users. This could be the case in a community focused on release discussions of Books, Movies, TV Shows or communities with content some users might find offensive but which is not neccesarily NSFW (for example jokes about certain topics).
+
+Essentially the tags serve as an easier way to message what is *inside* the post. Combining multiple tags allows for even finer control. Made a post about news in poland? Add the "News" and "Poland" tags to it and people interested exclusively in polish news can find it easily or people not interested can avoid it just as easily.
```

Comment:
I think a description of who defines a post's tags and who can edit them is missing

Reply:
Added some clarification around that (see https://github.com/LemmyNet/rfcs/commit/d894b357ef2372ec56a9b3e75fa95bc5d315ac94).